module.nix

```diff
@@ -17,6 +17,17 @@ in
       '';
       default = [ ];
     };
+    extraPostVM = lib.mkOption {
+      type = lib.types.str;
+      description = ''
+        extra shell code to execute once the disk image(s) have been succesfully created and moved to $out
+      '';
+      default = "";
+      example = pkgs.literalExpression ''
```

Comment:
This doesn't evaluate - `literalExpression` is in `lib`, not in `pkgs`

Reply:
Thank you! Fixed in https://github.com/nix-community/disko/pull/646